index.js

```diff
@@ -61,14 +67,12 @@ metagen.formats.deepAMD = files => `define([
 ], function() {
     return ${deepify(deepKeys(files), files.map((file, i) => `arguments[${i}]`))};
 });`
+metagen.formats.deepES6 = files => `${
+    files.map(file => `import ${varName(file)} from './${noExt(file)}';`).join('\n')
+}
+export default ${deepify(deepKeys(files), files.map(varName)).replace(/"/g, '').replace(/\n\s{4}/g, '\n')}`
```

Comment:
We should pull out an `unindent` function here like `indent`, although it is also worth asking why we need to unindent here in the first place